index.html

```diff
@@ -9,8 +9,9 @@
       href="https://www.finn.no/favicon.ico"
       type="image/x-icon"
     />
-    <link rel="stylesheet" href="https://assets.finn.no/pkg/@warp-ds/css/1.0.0-alpha.33/tokens/finn-no.css" />
-    <link rel="stylesheet" href="https://assets.finn.no/pkg/@warp-ds/css/1.0.0-alpha.33/resets.min.css" />
+    <link rel="stylesheet" href="https://assets.finn.no/pkg/@warp-ds/css/v1/tokens/finn-no.css" />
```

Comment:
If we use tokens from v1 (1.0.0-alpha.34) and component-classes from `1.0.0-alpha.37`, changes from `1.0.0-alpha.36` won't be applied due to missing tokens. It's just for internal testing of this repo but maybe until we release 1.0.0 and update the aliased version of @warp-ds/css we should continue referencing specific versions of eik urls. Wdyt?

Reply:
ah, my bad! Will fix!